subprojects/configuration-cache/build.gradle.kts

```diff
@@ -26,7 +26,7 @@ tasks.processResources {
 
 // The integration tests in this project do not need to run in 'config cache' mode.
 tasks.configCacheIntegTest {
-    enabled = false
+    enabled = true
```

Comment:
You definitely don't need this one. Gradle has few "flavors" of the integration tests, running in different modes (`--configuration-cache`, `--parallel`, `--no-daemon` and some others), in particular `configCacheIntegTest` means "run all tests with the configuration cache enabled". This makes sense for other parts of the codebase, but tests in the `configuraiton-cache` subproject manage configuration cache status themselves, so blanket "enable" is meaningless. That's why you see `enabled = false` here.

Reply:
got it! reverted